Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -308,10 +313,17 @@ final class CacheAdvanceTests: XCTestCase {
             let cachedMessages = try cache.messages()
 
             let secondCache = try createCache(overwritesOldMessages: true, maximumByteDivisor: maximumByteDivisor)
+
+            // Check that we can retrieve the messages from the first cache.
+            XCTAssertGreaterThan(try secondCache.messages().count, 0)
```

Comment:
`XCTAssertTrue(try secondCache.isEmpty())` may be more semantic, but this absolutely works. Thanks for adding!

Reply:
https://github.com/dfed/CacheAdvance/pull/45/commits/5a5f0077563719459e942e10eb09ca5f4a035ad7